drivers/misc/microbit_display/Kconfig

```
@@ -0,0 +1,93 @@
+# Microbit display driver configuration options
+
+# Copyright (c) 2017 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2021 Peter Niebert <peter.niebert@univ-amu.fr>
+# SPDX-License-Identifier: Apache-2.0
+
+config MICROBIT_DISPLAY
+	bool "BBC micro:bit 5x5 LED Display support"
+	depends on (BOARD_BBC_MICROBIT || BOARD_BBC_MICROBIT_V2)
+	help
+	  Enable this to be able to display images and text on the 5x5
+	  LED matrix display on the BBC micro:bit.
+
+if MICROBIT_DISPLAY
+
+config MICROBIT_DISPLAY_STR_MAX
+	int "Maximum length of strings that can be shown on the display"
+	range 3 255
+	default 40
+	help
+	  This value specifies the maximum length of strings that can
+	  be displayed using the mb_display_string() and mb_display_print()
+	  APIs.
+
+config MICROBIT_DISPLAY_TIMER_PRIORITY
+	int "NRFX Timer Interrupt priority"
+	default 1
+	help
+		Priority of the refresh interrupt of the
+		LED matrix. Can probably be low as it is not
+		critical.
+
+
+
+choice MICROBIT_DISPLAY_TIMER
+		prompt "NRFX Hardware Timer"
+		default MICROBIT_DISPLAY_TIMER3
+		help
+			Select the hardware timer instance used by this driver.
+			Beware of conflicts with other drivers, e.g. for bluetooth.
+
+# make sure the chosen timer is activated
+
+config MICROBIT_DISPLAY_TIMER0
+		bool "TIMER0"
+		depends on HAS_HW_NRF_TIMER0
```

I think it would be easier to avoid conflicts if you used `TIMERX_RESERVED` config entries.
```suggestion
		depends on HAS_HW_NRF_TIMER0
		depends on !NRF_HW_TIMER0_RESERVED
```